modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/package.json

```diff
@@ -2,7 +2,6 @@
 	"name": "frontend-taglib-clay-sample-web",
 	"private": true,
 	"scripts": {
-		"build": "liferay-npm-scripts build",
```

Comment:
I'm turning off any build here because I don't want babel or the bundler to rewrite anything.